lib/observability/tracing/collector.go

```diff
@@ -31,6 +31,8 @@ import (
 	"google.golang.org/grpc/credentials"
 	"google.golang.org/grpc/credentials/insecure"
 	"google.golang.org/protobuf/proto"
+
+	apidefaults "github.com/gravitational/teleport/api/defaults"
```

Comment:
Current imports of this `defaults` all seemed to be consistently aliasing to ` apidefaults`.  So I was just trying to stay consistent to that existing usage.  Let me know your thoughts

Reply:
IMO having the same aliases across all files makes things simpler. Otherwise, if `lib/defaults` is added to these files, we may end up with `libdefaults` and `defaults`, which points to something completely different than you would expect.